fixups/FileRedirectionFixup/PathRedirection.cpp

```diff
@@ -1201,6 +1201,12 @@ static path_redirect_info ShouldRedirectImpl(const CharT* path, redirect_flags f
     }
     LogString(inst, L"\tFRF Should: for path", widen(path).c_str());
     
+    size_t found = (widen(path)).find(L"WritablePackageRoot", 0);
+    if (found != 0)
+    {
+        LogString(inst, L"Prevent inception", widen(path).c_str());

```

Comment:
NIT: Add "Because writable package root was found." Or something that indicates why the redirection did not happen.

Reply:
Updated